gax/src/main/java/com/google/api/gax/tracing/ApiTracer.java

```diff
@@ -116,6 +116,19 @@
    */
   void attemptPermanentFailure(Throwable error);
 
+  /**
+   * Signals that the initial RPC for the long running operation failed.
+   *
+   * @param error the error that caused the long running operation fail.
+   */
+  void lroStartFailed(Throwable error);
```

Comment:
Unfortunately this will conflict and be confused with the existing methods of `operationSucceeded` which signal that the logical operation has completed